core/src/main/java/org/kiwix/kiwixmobile/core/history/HistoryActivity.java

```diff
@@ -94,27 +100,34 @@ public boolean onPrepareActionMode(ActionMode mode, Menu menu) {
     public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
       refreshAdapter = false;
       if (item.getItemId() == R.id.menu_context_delete) {
-        fullHistory.removeAll(deleteList);
-        for (HistoryListItem history : deleteList) {
-          int position = historyList.indexOf(history);
+        alertDialogShower.show(KiwixDialog.DeleteHistoryDialog.INSTANCE,
```

Comment:
We don't need the last `Dialog` on `DeleteHistoryDialog`, it goes against the naming convention of the rest of the dialogs in KiwixDialog and we know it is a dialog because it is prefixed by `KiwixDialog` and DialogShower only accepts the type `KiwixDialog`